app/konnect/dedicated-cloud-gateways.md

```diff
@@ -0,0 +1,31 @@
+---
+title: About Cloud Gateways
+content_type: concept
+---
+
+Dedicated cloud gateways are a type of API gateway in {{site.konnect_short_name}} that are completely managed by the cloud provider of your choice. You maintain control over the size and location of the gateway infrastructure, while Kong oversees the management of each instance and the entire cluster for you.
+
+You can manage your dedicated cloud gateways alongside all your other gateways in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
```

Comment:
```suggestion
You can manage your dedicated cloud gateway nodes alongside all your other gateway nodes in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
```
Maybe I'm wrong about the terminology here, but we're talking about data planes nodes/instances here, right?

Reply:
yes